### PR TITLE
Update README.md to add video links

### PR DESCRIPTION
**Pull request by stevevanhooser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ A MATLAB toolbox for accessing and using the neural recording public datasets fr
...
-| **Visual Behavior Neuropixels** [^5] |⚡ "ephys"<sup>b</sup> | Active<sup>d</sup> | [details](https://portal.brain-map.org/explore/circuits/visual-behavior-neuropixels) | TODO | TODO | 
+| Dataset | Recordings | Experiment | Details | View 👀 | Run ▶️ | Video |
+| --- | --- | --- | --- | --- | --- | --- |
+| **Visual Coding 2P** [^2] | 🔬 "ophys"<sup>a</sup> | Passive<sup>c</sup> | [details](http://portal.brain-map.org/explore/circuits/visual-coding-2p) | [![View Brain-Observatory-Toolbox on File Exchange](https://www.mathworks.com/matlabcentral/images/matlab-file-exchange.svg)](https://www.mathworks.com/matlabcentral/fileexchange/90900-brain-observatory-toolbox)  |    [![Open in MATLAB Online](https://www.mathworks.com/images/responsive/global/open-in-matlab-online.svg)](https://matlab.mathworks.com/open/github/v1?repo=MATLAB-Community-Toolboxes-at-INCF/Brain-Observatory-Toolbox&file=examples/VisualCoding_OphysQuickstart.mlx) | [video](https://www.youtube.com/watch?v=6pIWRXjHkNo) |
```
[the remark this answers is not in the document]
Comment (vijayiyer05) [reply README.md:26]: I agree with that. Besides being visually appealing/enticing, it helps make table narrower (better for smaller screens). 

I did check & there does not seem to be a YouTube badge for such purposes. So an emoji seems a/the good second best. I would likely use 🎞️ vs 🎬.

**Comment by vijayiyer05** [comment]
I will take a closer look this evening or tomorrow AM.
At first glance: the tables weren't rendering correctly in the Preview mode of the edited README.
I can empathize: it only takes a small slipup for table markdown to fail!

**Comment by ehennestad** [comment]
I should have fixed the readme, could you verify @vijayiyer05

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (stevevanhooser) [reply README.md:26]: What is the code for that filmstrip? I actually had a hard time figuring out what the film strip was whereas the clapper was obvious. Maybe it's the zoom level on my system or my eyes but the clapper seems more readable IMHO.

[the remark this answers is not in the document]
Comment (vijayiyer05) [reply README.md:26]: If you're curious, it's `:film_strip:`
Good perspective re immediate obviousness & readability (it's darker for sure).
Defer to your judgment!

**Pull request merged**